src/providers/delimitedtext/qgsdelimitedtextfeatureiterator.cpp

```diff
@@ -431,13 +431,25 @@ QgsGeometry QgsDelimitedTextFeatureIterator::loadGeometryXY( const QStringList &
     isNull = true;
     return QgsGeometry();
   }
+
   isNull = false;
-  QgsPointXY pt;
-  bool ok = QgsDelimitedTextProvider::pointFromXY( sX, sY, pt, mSource->mDecimalPoint, mSource->mXyDms );
+  QgsPoint *pt = new QgsPoint();
```

Comment:
```suggestion
  std::unique_ptr<QgsPoint> pt = qgis::make_unique<QgsPoint>();
```
for good practice and so on ;)